src/api/schema/broker.rs

```diff
@@ -0,0 +1,67 @@
+use futures::channel::mpsc::{self, UnboundedReceiver, UnboundedSender};
+use futures::task::{Context, Poll};
+use futures::{Stream, StreamExt};
+use once_cell::sync::Lazy;
+use serde::export::PhantomData;
+use slab::Slab;
+use std::any::{Any, TypeId};
+use std::collections::HashMap;
+use std::pin::Pin;
+use std::sync::Mutex;
+
+/// Current subscriptions. This is static so that 'global' changes can be published to any
+/// connecting subscriber. Subscribers fall out of scope once the subscription is terminated
+static SUBSCRIBERS: Lazy<Mutex<HashMap<TypeId, Box<dyn Any + Send>>>> = Lazy::new(Default::default);
```

Comment:
It looks like you're making an Anymap? https://docs.rs/anymap/0.12.1/anymap/

Reply:
Thanks! Switched to `anymap` in c672a42dac93e2ea0d2ee7db24cf412e9d73c0ce